corehq/apps/cloudcare/static/cloudcare/js/form_entry/fullform-ui.js

```diff
@@ -571,11 +582,20 @@ hqDefine("cloudcare/js/form_entry/fullform-ui", function () {
             return (self.error() || self.serverError()) && !self.dirty();
         });
 
+        self.form = function () {
```

Comment:
I'm surprised there wasn't already a way to get at the form inside `webformsession`, but I couldn't find one either.